### DISCUSSION TITLE
Why the default shell check?

### DISCUSSION THREAD

**Discussion by danb35** [tteck/Proxmox#3051] [General] [answered]
It seems that all the scripts have recently changed to check for Bash as the default shell, and abort if it isn't set.  Why would this be?  We're explicitly executing the scripts using Bash (e.g., `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/wikijs.sh)"`), and the scripts themselves also call for bash.  Why does it matter what my default shell is, if I'm specifically running the script under a different shell?

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/2536
